tips/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.utils import timezone
+from expenses.models import Expenses
+
+
+class Tip(models.Model):
+    author = models.CharField(max_length=32)
+    text = models.TextField(max_length=200)
+    date = models.DateTimeField(default=timezone.now)
+
+    category = models.CharField(
+         max_length=32,
+         choices=Expenses.Category.choices,
```

Comment:
We can make it a shared models.TextChoices
Can you do it?

Reply:
Working on it!

Reply:
I think that Shelly suppose to gennerelize the Category enum class so we could use it in different apps.
@ShellyGolden

Reply:
Any update about that? we want to merge it as fast as possible

Reply:
Or said in the slack that he will fix it tonight 
@ronyyosef

Reply:
I think it could be merged right now or if you think the generalisation is critical you can do it right now according to doc I‘ve sent in the slack I will be available to do this only at 23:00 
@ronyyosef

Reply:
@orIsrael 
Leave it like this, if we will have later on more shared things between different apps we will open a shared directory and pull the things from one shared location